src/sys/mod.rs

```diff
@@ -47,3 +47,10 @@ pub mod time;
               target_arch = "arm")),
           )]
 pub mod ptrace;
+
+#[cfg(all(target_os = "linux",
+          any(target_arch = "x86",
+              target_arch = "x86_64",
+              target_arch = "arm")),
+          )]
+pub mod quotactl;
```

Comment:
This should probably be named `quota` as the corresponding C header file is `sys/quota.h`.